### PR TITLE
Add weth feed to ccip deployment tests

### PR DESCRIPTION
**Pull request by asoliman92**

### CODE REVIEW

File: integration-tests/deployment/ccip/add_chain_test.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
 )
 
-func TestAddChainInbound(t *testing.T) {
+func TestAddChain(t *testing.T) {
```
Comment (AnieeG) [inline integration-tests/deployment/ccip/add_chain_test.go:32]: If you are enabling the chain both way - let's send message in other direction to see whether that's successful

File: integration-tests/deployment/ccip/add_chain_test.go [hunk 2]
```diff
@@ -224,9 +235,8 @@ func TestAddChainInbound(t *testing.T) {
 	require.NoError(t,
 		ConfirmExecWithSeqNr(t, e.Env.Chains[initialDeploy[0]], e.Env.Chains[newChain], state.Chains[newChain].OffRamp, &startBlock, seqNr))
 
-	linkAddress := state.Chains[newChain].LinkToken.Address()
-	feeQuoter := state.Chains[newChain].FeeQuoter
-	timestampedPrice, err := feeQuoter.GetTokenPrice(nil, linkAddress)
-	require.NoError(t, err)
-	require.Equal(t, MockLinkPrice, timestampedPrice.Value)
+	// TODO: Properly assert by checking events
```
Comment (AnieeG) [inline integration-tests/deployment/ccip/add_chain_test.go]: create a ticket please

File: integration-tests/deployment/ccip/test_helpers.go [hunk 3]
```diff
@@ -458,49 +467,230 @@ const (
...
+	}
+
+	linkFeedAddress, linkFeedDescription, err := deploySingleFeed(lggr, ab, chain, mockLinkFeed, LinkSymbol)
 	if err != nil {
-		lggr.Errorw("Failed to deploy link feed", "err", err)
 		return nil, err
 	}
 
-	lggr.Infow("deployed mockLinkFeed", "addr", mockLinkFeed.Address)
+	wethFeedAddress, wethFeedDescription, err := deploySingleFeed(lggr, ab, chain, mockWethFeed, WethSymbol)
```
Comment (AnieeG) [inline integration-tests/deployment/ccip/test_helpers.go]: errcheck

File: integration-tests/deployment/ccip/test_helpers.go [hunk 4]
```diff
@@ -458,49 +467,230 @@ const (
...
 
-	desc, err := mockLinkFeed.Contract.Description(&bind.CallOpts{})
+func deploySingleFeed(
+	lggr logger.Logger,
+	ab deployment.AddressBook,
+	chain deployment.Chain,
+	deployFunc func(deployment.Chain) ContractDeploy[*aggregator_v3_interface.AggregatorV3Interface],
+	symbol TokenSymbol,
+) (common.Address, string, error) {
+	//tokenTV := deployment.NewTypeAndVersion(PriceFeed, deployment.Version1_0_0)
```
Comment (AnieeG) [inline integration-tests/deployment/ccip/test_helpers.go]: remove?

File: integration-tests/deployment/ccip/token_info.go [hunk 5]
```diff
@@ -28,10 +29,11 @@ func (tc *TokenConfig) UpsertTokenInfo(
 	tc.TokenSymbolToInfo[symbol] = info
 }
 
-// GetTokenInfo Adds mapping between dest chain tokens and their respective aggregators on feed chain.
-func (tc *TokenConfig) GetTokenInfo(
+// GetLinkInfo Adds mapping between dest chain tokens and their respective aggregators on feed chain.
+func (tc *TokenConfig) GetLinkInfo(
```
Comment (AnieeG) [inline integration-tests/deployment/ccip/token_info.go]: you are also sending Weth

**Review by winder — DISMISSED** [review]
Lets get this into fix-fee-quoter asap so that CI can run

**Comment by AnieeG** [comment]
created a new one https://github.com/smartcontractkit/chainlink/pull/14904

**Pull request closed without merging**